utility/sort/sort_test.go

```diff
@@ -0,0 +1,40 @@
+package sort
+
+import "github.com/stretchr/testify/suite"
+import (
+	"github.com/gojektech/proctor/proctord/jobs/metadata"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+type SortTestSuite struct {
```

Comment:
@RajatVaryani we use suites only for removing duplication of multiple setups and teardowns. Could you use a simple test here instead?

Reply:
Made changes.